pyanaconda/network.py

```diff
@@ -203,6 +203,7 @@ def getHostname():
 
     return hn
 
+# TODO: this function is never used
```

Comment:
@rvykydal aren't we doing the same somewhere else? It think I have seen output similar to this.

Reply:
I think we can remove it. We are using logIfcfgFiles everywhere now.

Reply:
Removed.